CHANGELOG.md

```diff
@@ -4,9 +4,13 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- A new `UVCal.new()` method (based on new function `new_uvcal`) that creates a new,
+self-consistent `UVCal` object from scratch from a set of flexible input parameters.
```

Comment:
Let's add something about the new `docstring_parser` dependency here. It will help me when we make the next release as well as helping users.